### DISCUSSION TITLE
Telegram alerts not working

### DISCUSSION THREAD

**Discussion by nhasian** [semaphoreui/semaphore#2135] [Q&A] [answered]
Using ansible-semaphore in docker with image semaphoreui/semaphore:latest
I used the @rawdatabot in Telegram to retrieve my `chat id`.
In ansible-semaphore from the Dashboard -> SETTINGS, I enter my `chat id` in "Telegram Chat ID (Optional) and click SAVE.
I'm not sure if it matters for Telegram, but I also have `Send alerts` enabled for the admin user, though I suspect this is just for email notifications.

When running a task, whether it is Success or Failed, I do not receive any Telegram notifications.

**Comment by dzonder** [comment]
I think one needs to set the following in the Semaphore config:

```
telegram_alert: true
telegram_chat: <chat id>
telegram_token: <token from BotFather>
```

However so far I also haven't been successful with sending alerts - a failure is reported upon a failing task:

```
7:48:56 PM
Running playbook failed: exit status 4
7:48:56 PM
Can't send telegram alert! Response code: 403
```

So at least I know that Semaphore tries to send the alert. Not clear to me why 403 error is triggered.

**Comment by prohand** [comment]
Hello, same problem for me :(
Do you have a solution please ?
Thank you

**Comment by fiftin** [comment]
Telegram alerts work. It is configuration issue. Did you test your bot via curl?

**Reply by prohand** [reply]
I have this error : 
![image](https://github.com/semaphoreui/semaphore/assets/20227162/cafe84e6-a7dc-472c-9ce9-9f66f9636ad3)

I check

**Reply by prohand** [reply, marked as answer]
I have solved my problem.
I create a groupe with ansible bot and copy the chat_id of group and it's ok ;)

Thank you

**Reply by cbugk** [reply]
I had my profile private so was getting a 400 error. This work-around allowed me to keep my account private.
